pkg/cmd/server/origin/master_config.go

```diff
@@ -318,6 +318,7 @@ var (
 		"SCCExecRestrictions",
 		quotaadmission.PluginName,
 		"ClusterResourceQuota",
+		"PersistentVolumeLabel",
```

Comment:
Seems like a comment is in order for that so the next poor soul doesn't do this too